src/features/workflow/WorkflowCell.tsx

```diff
@@ -91,43 +91,42 @@ const WorkflowCell: React.FC<WorkflowCellProps> = ({
   }
 
   return (
-    <div id={`${name}-cell`} className={`w-full workflow-cell flex ${isCellAdded && 'animate-appear'}`} onClick={onClick}>
-      <div className='flex-grow min-w-0'>
+    <div id={`${name}-cell`} className={`w-full workflow-cell relative  ${isCellAdded && 'animate-appear'}`} onClick={onClick}>
         <ScrollAnchor id={name} />
+        <div
+          className={classNames(`border rounded-lg ${active && borderStyles}`, {
+            'border-transparent': !active
+          })}
+          style={{
+            borderRadius: '0.58rem',
+            width: 'calc(100% - 225px)'
+          }}
+        >
+          {/* ^^ this wrapping div allows us to use two different borders.  Active cell shows both, allowing for a thicker border without causing content to shift*/}
           <div
-            className={classNames(`border rounded-lg ${active && borderStyles}`, {
-              'border-transparent': !active
+            className={classNames(`rounded-lg overflow-hidden flex-grow border box-content ${borderStyles}`, {
+              'border-transparent hover:border-qrigray-200': !active && (noColorStatus)
             })}
```

Comment:
this can be rewritten to drop the template string (which transpiles to a function call IIRC):

```ts
className={classNames(
  'rounded-lg overflow-hidden flex-grow border box-content',
  borderStyles,
  'border-transparent hover:border-qrigray-200': !active && (noColorStatus)
)}
```